src/app/functions/server/updateDataBrokerScanProfile.ts

```diff
@@ -64,16 +64,37 @@ async function updateDataBrokerScanProfile(
     }
   }
 
+  const updatedProfileDataSanitized = {
+    first_name: updatedProfileData.first_name.trim(),
+    last_name: updatedProfileData.last_name.trim(),
+    first_names: updatedProfileData.first_names
+      .filter((value) => value.trim())
+      .map((first_name) => first_name.trim()),
+    last_names: updatedProfileData.last_names
+      .filter((value) => value.trim())
+      .map((last_name) => last_name.trim()),
+    middle_names: updatedProfileData.middle_names
+      .filter((value) => value.trim())
+      .map((middle_name) => middle_name.trim()),
+    phone_numbers: updatedProfileData.phone_numbers
+      .filter((value) => value.trim())
```

Comment:
Issue: I think these were the cause of the errors I was getting - if I submit the form without entering data into one of the form fields, there's an error on the server, and on the frontend I only see my data being thrown away. Can't call `.trim()` on `undefined` or something like that.

(Which also makes me wonder: if there a type definition somewhere that's too loose?)

Reply:
OK, I can't reproduce anymore apparently :/ Maybe it had to do with my just having entered my details on the welcome screen, but otherwise I guess this one can be resolved.